openpype/hosts/blender/api/ops.py

```diff
@@ -344,6 +345,91 @@ def before_window_show(self):
         self._window.refresh()
 
 
+class WM_OT_DownloadLastWorkfile(bpy.types.Operator):
+    """Download Last Workfile."""
+
+    bl_idname = "wm.download_last_workfile"
+    bl_label = "Download Latest Workfile"
+
+    @classmethod
+    def poll(cls, context):
+        """Return if this operator is available or not."""
+        return context.window_manager.is_workfile_out_of_date
+
+    def execute(self, context):
+        """Execute this operator."""
+        # Get sync server module
+        sync_server = ModulesManager().modules_by_name.get("sync_server")
+        if not sync_server or not sync_server.enabled:
+            self.report(
+                {"WARNING"}, "Sync server module is disabled or unavailable."
+            )
+            return {"CANCELLED"}
+
+        last_workfile_path = download_last_workfile()
```

Comment:
I think `download_last_workfile` must be changed to `get_last_workfile_representation(project_name, asset_name, task_name)` where you'd put almost everything from `download_last_workfile` instead of the last call `download_last_published_workfile`. This last call should be put right after using the gotten representation.

```suggestion
        session = legacy_io.Session
        project_name = session.get("AVALON_PROJECT")
        task_name = session.get("AVALON_TASK")
        asset_name = session.get("AVALON_ASSET")
        anatomy = Anatomy(project_name)
        last_workfile_representation = get_last_workfile_representation(project_name, asset_name, task_name)
        last_workfile_path = download_last_published_workfile(...)
```